101-servicebus-create-namespace/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceBusNamespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Service Bus namespace"
+            }
+        },
+        "serviceBusSku": {
+            "type": "string",
+            "allowedValues": [
+                "Basic",
+                "Standard"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "The messaging tier for service Bus namespace"
+            }
+        },
+        "serviceBusApiVersion": {
```

Comment:
In that case this parameter needs a default value. Here's how it looks now

![image](https://cloud.githubusercontent.com/assets/6164178/14571663/4efba6de-02ff-11e6-9d6c-1ec8e798b885.png)